evap/staff/templates/staff_user_index.html

```diff
@@ -82,7 +81,7 @@
 {% block modals %}
     {{ block.super }}
     {% trans 'Delete user' as title %}
-    {% trans 'Do you really want to delete the user <strong id="data-label"></strong>?' as question %}
+    {% trans 'Do you really want to delete the user <strong id="data-label"></strong>?<br><strong id="data-label"></strong> will also be removed from every user having <strong id="data-label"></strong> as a delegated or CC-user.' as question %}
```

Comment:
please change to `{% trans 'Do you really want to delete the user <strong id="data-label"></strong>?<br />This person will also be removed from every other user having this person as a delegated or CC-user.' as question %}`